supporting_scripts/course-scripts/quick-course-setup/README.md

```diff
@@ -0,0 +1,138 @@
+
+1. Start your local Artemis instance.
+
+2. Configure the values in config.ini according to your setup.
+
+3. Install the missing packages of the Python scripts that you want to execute (if not done already).
+
+4. Run the main script using IntelliJ:
+
+   •	Use the play button within IntelliJ (if Python is configured properly).
+   •	Alternatively, follow the command line instructions for each script.
+
+### Creating a Course with Standard User Groups
+
+Creates a course for which the users are registered according to their user groups (students, tutors, editors, instructors).
+
+```shell
+python3 create_course.py
+```
+
+### Create Users
+
+Creates users 1-20 (students, tutors, editors, instructors - 5 for each group) and users needed for Cypress E2E testing (100-106).
+    
+```shell
+python3 create_users.py
+```
+
+### Authenticate Users
+
+If the users have already been created, they still need to be logged in order to be added to a course (without a first login, Artemis does not know that the users exist).
+    
+```shell  
+python3 authenticate_all_users.py
+```
+
+## Test Servers
+
+### To configure the scripts for use with Artemis test servers:
+
+1.	Adjust server_url and client_url according to the test server.
+2.  Update admin_user and admin_password to valid values for the test server.
+3.	Set is_local_course in config.ini to False.
+
+Create a Course and Assign the Default Users
+
+1-5: students
+6-10: tutors
+11-15: instructors
+16-20: editors
+
+Define the name of your course in the config.ini as course_name.
+
+```shell  
+python3 create_course.py
+```
+
+### Add Users to Existing Course
+
+Define the course_id in the config.ini.
+
+```shell  
+python3 add_users_to_course.py
+```
+
+## Notes
+
+1.  Ensure that the config.ini file is correctly configured before running any scripts.
+2.  Always test the scripts on a local setup before running them on a production or test server.
+
+## Section 2: Artemis Large Course with Submissions on Programming Exercise Automation
+
+This section details how to use the large_course_main script that orchestrates the entire process, from user creation to course setup, and finally user participation in programming exercises.
+
+### Running the large_course_main Script
+
+The large_course_main script performs all necessary steps to set up a large course with multiple students and submissions.
+
+### Steps to Run:
+
+1. Open the project in IntelliJ. 
+2. Locate the large_course_main.py file in the project directory.
+3. To change the number of students created, modify the students variable in the config.ini file.
+4. To change the number of commits each student should perform in the example exercise, modify the commits variable in the config.ini file.
+5. To change the number of programming exercises created, modify the exercises variable in the config.ini file.
+6. You can use the play button within IntelliJ (if Python is configured properly) to run the script. (Run large_course_main.py)
+
```

Comment:
Could we directly add the option to execute the script here? :) 
```suggestion
6. You can use the play button within IntelliJ (if Python is configured properly) to run the script. (Run large_course_main.py)
   ```shell
    python3 large_course_main.py
    ```
```